src/rsz/src/Resizer.cc

```diff
@@ -2855,6 +2850,43 @@ void Resizer::rebufferNet(const Pin* drvr_pin)
 
 ////////////////////////////////////////////////////////////////
 
+OperationSettings::OperationSettings(bool* exclude_clock_buffers,
+                                     const bool new_exclude_clock_buffers,
+                                     sta::LibertyCellSeq* buffer_cells)
+{
+  if (exclude_clock_buffers) {
+    exclude_clock_buffers_ = exclude_clock_buffers;
+    prev_exclude_clock_buffers_ = *exclude_clock_buffers_;
+    *exclude_clock_buffers_ = new_exclude_clock_buffers;
+  }
+
+  if (buffer_cells) {
+    buffer_cells_ = buffer_cells;
+    buffer_cells_->clear();
```

Comment:
Why set and then clear?

Reply:
The idea would be that for this use case we only clear the RSZ buffer list for the subsequent preamble to populate it.
We need to set the pointer so that the destructor can clear it again for the next operation.